pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1457,6 +1457,37 @@ class Timestamp(_Timestamp):
         tz = maybe_get_tz(tz)
         return cls(datetime.fromtimestamp(ts, tz))
 
+    def fast_strftime(self, fmt_str: str, loc_s: object) -> str:
+        """A faster alternative to `strftime` using string formatting.
```

Comment:
any reason why there's still a new function?

In https://github.com/pandas-dev/pandas/pull/51298#issuecomment-1655952627 I'd suggest to add a hypothesis test - if we're confident in it, let's just use it and speed up where possible

Reply:
This instance-related one is a low-level utility function for internal calls. Its inputs are the outputs of the string template converter, so it is only for debugging purposes. If you prefer I can add an underscore prefix for this one.

> I'd suggest to add a hypothesis test - if we're confident in it, let's just use it and speed up where possible

Is the current hypothesis test not enough ? I had inserted it in response to that old comment of yours. https://github.com/pandas-dev/pandas/pull/51298/files#diff-119789a2741b06c31031dfb70a5cbd9aab84d854dec9b649e2fd67c76ff00531R970